kafka/consumer/main.go

```diff
@@ -251,11 +261,31 @@ func (cgh *consumerGroupHandler) ConsumeClaim(session sarama.ConsumerGroupSessio
 	}
 }
 
-func inClusterKafkaConfig() (kafkaConfig *sarama.Config, err error) {
-	kafkaConfig = sarama.NewConfig()
-	kafkaConfig.Consumer.Offsets.Initial = sarama.OffsetNewest
-	kafkaConfig.Version = sarama.V0_10_2_1
-	return kafkaConfig, nil
+func getConfig() *sarama.Config {
```

Comment:
We should probably make a common/utils file or something and put this and the getEnv function in it since they're used in both the consumer and producer

Reply:
This is a bit complicated because consumer and producer resides in their individual dir and the vendor folder and go.mod file also resides in there. No matter where we put the getConfig func, we will have to treat it as an individual module and create another set of go.mod go.sum files due to the need of sarama lib. I just tested this, and it does not look very pretty. 
However, I did put getEnv in the utils folder.

Reply:
Update: because this demo project did not use the traditional golang dir structure of root/cmd and root/pkg. It's a bit complicated to refactor util. I put the util.go into pkg/util folder but had a lot of issues during build. The main.go of consumer and producer could not find it because neither of these files exist under root/cmd folder. So it would be a bit involved to refactor. I suggest treating it as a separate PR.